components/GridCard/index.tsx

```diff
@@ -64,43 +72,28 @@ const Card = ({
   );
 };
 
-export const AssetCard = ({
-  name,
-  asset_id,
-  data: { image },
-  isForSale,
-  salePrice,
-  template_mint,
-  template: { max_supply },
-}: Asset): JSX.Element => {
-  return (
-    <Card
-      image={image as string}
-      text={name}
-      secondaryText={`Serial #${template_mint}/${max_supply}`}
-      priceText={salePrice as string}
-      isForSale={isForSale as boolean}
-      redirectPath={`/assets/${asset_id}`}
-    />
-  );
-};
-
 export const TemplateCard = ({
   name,
   max_supply,
   template_id,
   immutable_data: { image },
   lowestPrice,
   isLoading,
+  assetsForSale,
+  totalAssets,
+  isUsersTemplates,
 }: TemplateCardProps): JSX.Element => {
   return (
     <Card
       isLoading={isLoading}
       text={name}
-      secondaryText={`Edition Size / ${formatNumber(max_supply)}`}
+      secondaryText={`Edition Size: ${formatNumber(max_supply)}`}
       priceText={lowestPrice}
       image={image as string}
       redirectPath={`/${template_id}`}
```

Comment:
Can we conditionally set the `redirectPath` (`/my-templates/${template_id}` when `isUsersTemplates` is true)?